### DISCUSSION TITLE
React Native Support

### DISCUSSION THREAD

**Discussion by 321paranoiawhy** [barvian/number-flow#7] [Ideas]
Would you like to support react native？Of course I wanna have a try but I have to read the source code, that is very heavy work for me. And what do you think of？ Thanks your great work.

**Comment by barvian** [comment]
Hey, I'd love to support React Native but I don't have any experience with it or extra bandwidth right now. I'd happily support anybody who wants to take a stab it though! Converting this to a discussion for now.

**Comment by Rednegniw** [comment]
Hi, just seconding this, a React Native version of this would be amazing.

**Reply by 321paranoiawhy** [reply]
You can try rn-number-flow
